xbmc/settings/AdvancedSettings.h

```diff
@@ -344,6 +344,13 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     std::string m_cpuTempCmd;
     std::string m_gpuTempCmd;
 
+    // Touchscreen allignment advanced settings
+    int m_xOffset;		// x axis offset of touch panel coordinate origin
+    int m_yOffset;		// y axis offset of touch panel coordinate origin
+    float m_xStretchFactor;	// x axis stretch factor
+    float m_yStretchFactor;	// y axis stretch factor
+    int m_confines; 		// click or touch confines setting
+
```

Comment:
Better to have a more detailed name about the settings label (is a big list there) maybe "m_screenAlign_xOffset".

Also if on xbmc/input/MouseStat.h the description for m_confines description is "number of pixels that the pointer may move while the button is down to trigger a click" better to use the same here.